Cargo.toml

```diff
@@ -23,6 +25,7 @@ rayon = { version = "1.7.0", optional = true }
 giza-core = { git = "https://github.com/lambdaclass/giza", branch = "remove_prints", optional = true }
 giza-prover = { git = "https://github.com/lambdaclass/giza", branch = "remove_prints", optional = true }
 giza-runner = { git = "https://github.com/lambdaclass/giza", branch = "remove_prints", optional = true }
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
I think we're missing marking this as optional.

Reply:
It's ok, we Serde will be the only serialziation in our repo, we will remove our own. In that case it will have to be non optional